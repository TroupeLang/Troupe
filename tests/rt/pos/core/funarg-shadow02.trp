let fun f x y x = x in
  f 1 2 3
end