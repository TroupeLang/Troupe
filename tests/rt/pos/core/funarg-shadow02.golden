2023-03-10T08:32:11.371Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 3@{}%{}
